go.mod

```diff
@@ -1,4 +1,4 @@
-go 1.15
+go 1.16
```

Comment:
embarassingly, it's been so long that we actually don't build with anything less than 1.17 now. could you update it @rockmenjack ? also, thanks for your patience here